README.md

```diff
@@ -45,23 +51,25 @@ may also be helpful.
 
 ## Quick usage
 
+<!---
```

Comment:
I included `## Awecode` in the comment because the whole section is optional,
while I excluded `## Quick usage` from the comment because the section is mandatory.
the idea is to have 2 ways to detect missing text:

- by remaining comments (maybe we should reword them to `<!--- README`?)
- by previewing md -> empty paragraph